bin/plot_krun_results

```diff
@@ -451,8 +451,19 @@ class ProcessExecChart(object):
             self.plot_instr_data()
         if self.y_range_zoom[0] != None:
             self.plot_zoomed_data()
+        self._increment_xticklabels(self.wallclock_axis)
```

Comment:
Would it make more sense to pass the x-axis itself with 1-based indicies rather than fixing them up later?

so, in your call to matplotlib.pyplot.plot, you'd have something like:

```
plot(xrange(1,2001), ys)
```